react-cra/src/BUILD.bazel

```diff
@@ -52,6 +52,7 @@ ts_project(
     srcs = glob(TEST_PATTERNS),
     declaration = True,
     resolve_json_module = True,
+    supports_workers = False,
```

Comment:
Can we put a comment here? Even just moving the one deleted from ci.yaml...

Reply:
Will do